Core/BlockerListsLoader.swift

```diff
@@ -80,7 +81,15 @@ public class BlockerListsLoader {
             semaphore.signal()
         }
 
-        return 3
+        blockerListRequest.request(.httpsUpgrade) { (data) in
+            if let data = data {
+                self.newDataItems += 1
+                self.httpsUpgradeStore.persist(data: data)
+            }
+            semaphore.signal()
+        }
+
+        return 4
```

Comment:
What's this for?

Reply:
My original thinking was that this method would return the count of some time of operation array, but it didn't pan out that way.  Could just as easily be hardcoded in to the loop which this is called from.  Ideally should probably use an NSOperation, but I think I'll refactor that later.